opta/commands/destroy.py

```diff
@@ -51,6 +52,7 @@ def destroy(
     """
 
     config = check_opta_file_exists(config)
+    symlink_check()
```

Comment:
You don't need this in destroy

Reply:
destroy would also be needing the paths right? Or is that not path dependent.
If not, I'll revert this PR, and create a fresh one..

Reply:
If it's a symlink path it will get caught during the apply. So we can assume destroy won't run into this.
You don't need to revert the PR - just make a new one and remove this line from destroy.

Reply:
#472